packages/protocol/contracts/L1/TaikoConfig.sol

```diff
@@ -11,38 +11,44 @@ import { TaikoData } from "../L1/TaikoData.sol";
 library TaikoConfig {
     function getConfig() internal pure returns (TaikoData.Config memory) {
         return TaikoData.Config({
+            // Group 1: general configs
             chainId: 167,
+            relaySignalRoot: false,
+            // Group 2: block level configs
             // Two weeks if avg block time is 10 seconds
-            maxNumProposedBlocks: 120_960,
-            ringBufferSize: 120_960 + 10,
+            blockMaxProposals: 120_960,
+            blockRingBufferSize: 120_960 + 10,
             // Each time one more block is verified, there will be ~20k
             // more gas cost.
-            maxVerificationsPerTx: 10,
+            blockMaxVerificationsPerTx: 10,
             // Set it to 6M, since its the upper limit of the Alpha-2
             // testnet's circuits.
             blockMaxGasLimit: 6_000_000,
+            blockFeeBaseGas: 20_000,
+            // Set it to 79  (+1 TaikoL2.anchor transaction = 80),
+            // and 80 is the upper limit of the Alpha-2 testnet's circuits.
+            blockMaxTransactions: 79,
             // Set it to 120KB, since 128KB is the upper size limit
             // of a geth transaction, so using 120KB for the proposed
             // transactions list calldata, 8K for the remaining tx fields.
-            maxBytesPerTxList: 120_000,
-            proofCooldownPeriod: 30 minutes,
-            systemProofCooldownPeriod: 15 minutes,
-            // Only need 1 real zkp per 10 blocks.
-            // If block number is N, then only when N % 10 == 0, the real ZKP
-            // is needed. For mainnet, this must be 0 or 1.
-            realProofSkipSize: 10,
-            // Set it to 79  (+1 TaikoL2.anchor transaction = 80),
-            // and 80 is the upper limit of the Alpha-2 testnet's circuits.
-            maxTransactionsPerBlock: 79,
+            blockMaxTxListBytes: 120_000,
+            blockTxListExpiry: 0,
+            // Group 3: proof related configs
+            proofRegularCooldown: 30 minutes,
+            proofOracleCooldown: 15 minutes,
+            proofMinWindow: 10 minutes,
+            proofMaxWindow: 90 minutes,
```

Comment:
The two parameters are related to circuit size. If this circuit is tiny, we don't have to set a min window of 10min. The max window is similar.